rust_dev_preview/sitewise/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "sitewise-code-examples"
+version = "0.1.0"
+authors = ["Lawrence Xiao <delonx@amazon.com>"]
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
For more keys and their definitions, see https://doc.rust-lang.org/cargo/reference/manifest.html.

Reply:
updated